gh-134100: Fix UAF in PyImport_ImportModuleLevelObject (#134117)
python/cpython: Misc/NEWS.d/next/Core_and_Builtins/2025-05-16-17-25-52.gh-issue-134100.5-FbLK.rst

Fixed a use-after-free bug that would occur when an imported module wasn't

> in sys.modules after the initial import. Patch by Nico-Posada.

Reviewer: ```suggestion
in :data:`sys.modules` after the initial import. Patch by Nico-Posada.
```

> Fixed a use-after-free bug that would occur when an imported module wasn't
> in :data:`sys.modules` after the initial import. Patch by Nico-Posada.

Reviewer: The changelog is typically in present-tense:
```suggestion
Fix a use-after-free bug that occurs when an imported module isn't
in :data:`sys.modules` after its initial import. Patch by Nico-Posada.
```
